tests/unit/test_energy_spectrum.py

```diff
@@ -271,5 +255,181 @@ def test_when_no_foil_is_true(self):
+            path=tmp_dir / "mesu.info", save_on_creation=True)
+        es = self.get_energy_spectra
+        sum_spectrum = SumEnergySpectrum(
+            es, mesu.directory, SumSpectrumType.MEASURED)
+        return sum_spectrum, es, mesu
+
+    @property
+    def created_sum_spectrum(self):
+        return [
+                (-0.75, 0.0),
+                (-0.25, 0.0),
+                (0.25, 7.0),
+                (0.75, 11.0),
+                (1.25, 0.0),
+                (1.75, 2.0),
+                (2.25, 8.0),
+                (2.75, 3.0),
+                (3.25, 6.0),
+                (3.75, 0.0)]
+
+    @property
+    def sum_spectrum_after_removed_spectra(self):
+        return [
+                (-0.75, 0.0),
+                (-0.25, 0.0),
+                (0.25, 7.0),
+                (0.75, 1.0),
+                (1.25, 0.0),
+                (1.75, 2.0),
+                (2.25, 8.0),
+                (2.75, 3.0),
+                (3.25, 6.0),
+                (3.75, 0.0)]
+
+    @property
+    def sum_spectrum_with_added_spectra(self):
+        return [(-0.95, 0.0),
+                (-0.75, 0.0),
+                (-0.45, 0.0),
+                (-0.25, 1.5555555555555556),
+                (0.25, 12.444444444444443),
+                (0.45, 15.6),
+                (0.75, 14.399999999999999),
+                (0.95, 7.600000000000001),
+                (1.25, 0.3999999999999999),
+                (1.45, 0.7999999999999998),
+                (1.75, 2.0),
+                (2.25, 8.0),
+                (2.75, 3.0),
+                (3.25, 6.0),
+                (3.75, 0.0)]
+
+    @property
+    def new_spectra(self):
+        return {'1H.ERD.2': [
+            (-0.95, 0), (-0.45, 0.0), (0.45, 7.0), (0.95, 1.0), (1.45, 0)]
+        }
+
+    @property
+    def expected_sum_spectrum_file_content(self):
+
+        return [
+            '-0.75000      0',
+            '-0.25000      0',
+            '0.25000      7',
+            '0.75000     11',
+            '1.25000      0',
+            '1.75000      2',
+            '2.25000      8',
+            '2.75000      3',
+            '3.25000      6',
+            '3.75000      0'
+        ]
+
+    @property
+    def get_energy_spectra(self):
+        return {
+            '1H.ERD.0': [
+                (-0.25, 0), (0.25, 0.0), (0.75, 10.0), (1.25, 0)
+            ],
+            '1H.ERD.1': [
+                (-0.75, 0), (-0.25, 0.0), (0.25, 7.0), (0.75, 1.0), (1.25, 0)
+            ],
+            '35Cl.RBS_Mn.0': [
+                (1.75, 0), (2.25, 0.0), (2.75, 3.0), (3.25, 6.0), (3.75, 0)
+            ],
+            '7Li.0.0.0': [
+                (0.75, 0), (1.25, 0.0), (1.75, 2.0), (2.25, 8.0), (2.75, 0)
+            ]
+        }
+
+    def test_sum_spectrum_creation(self):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+
+            sum_spectrum, es, _ = self.create_sum_and_energy_spectrum(tmp_dir)
+
+            # Check if a created sum spectrum is correct
```

Comment:
A comment like would work better as the docstring for the test method. The same applies for the other tests' comments.